packages/hoppscotch-common/src/newstore/settings.ts

```diff
@@ -30,6 +30,31 @@ export type SettingsDef = {
 
   PROXY_URL: string
 
+  // HTTP Wrap Lines
+  WRAP_LINES_HTTP_REQUEST_BODY: boolean
+  WRAP_LINES_HTTP_RESPONSE_BODY: boolean
+  WRAP_LINES_HTTP_HEADERS: boolean
+  WRAP_LINES_HTTP_PARAMS: boolean
+  WRAP_LINES_HTTP_URL_ENCODED: boolean
+  WRAP_LINES_HTTP_PREREQUEST: boolean
+  WRAP_LINES_HTTP_TEST: boolean
+
+  // GraphQL Wrap Lines
+  WRAP_LINES_GRAPHQL_QUERY: boolean
+  WRAP_LINES_GRAPHQL_RESPONSE_BODY: boolean
+  WRAP_LINES_GRAPHQL_HEADERS: boolean
+  WRAP_LINES_GRAPHQL_VARIABLES: boolean
+  WRAP_LINES_GRAPHQL_SCHEMA: boolean
+
+  // Realtime Wrap Lines
+  WRAP_LINES_REALTIME_BODY: boolean
+  WRAP_LINES_REALTIME_LOG: boolean
+
+  // Other Wrap Lines
+  WRAP_LINES_IMPORT_CURL: boolean
+  WRAP_LINES_CODE_GEN: boolean
+  WRAP_LINES_COOKIE: boolean
```

Comment:
Can we group these under a property, say `WRAP_LINES` similar to `URL_EXCLUDES? It might require changes to the [toggleSetting function](https://github.com/hoppscotch/hoppscotch/blob/main/packages/hoppscotch-common/src/newstore/settings.ts#L138-L145) and the corresponding [settings store dispatcher](https://github.com/hoppscotch/hoppscotch/blob/main/packages/hoppscotch-common/src/newstore/settings.ts#L87-L102) since it only accepts a top-level setting key currently.

Reply:
Okay, I'll change that

Reply:
> Can we group these under a property, say `WRAP_LINES` similar to `URL_EXCLUDES`? It might require changes to the [toggleSetting function](https://github.com/hoppscotch/hoppscotch/blob/main/packages/hoppscotch-common/src/newstore/settings.ts#L138-L145) and the corresponding [settings store dispatcher](https://github.com/hoppscotch/hoppscotch/blob/main/packages/hoppscotch-common/src/newstore/settings.ts#L87-L102) since it only accepts a top-level setting key currently.

I've created seperate functions instead of extending the current ones, parsing by `.` and changing state from the current function doesn't provide flexibility for proper type checking